microsoft-r/operationalize/how-to-create-manage-session-pools.md

```diff
@@ -37,7 +37,7 @@ For R script, the [mrsdeploy](../r-reference/mrsdeploy/mrsdeploy-package.md) fun
 
 ## Create or modify a dedicated session pool
 
-You can use an R console application, such as Rgui.exe, to run the following commands on the local server if you configured Machine Learning Server for one-box, or on a compute node if you have a distributed topology.
+An already stablished connection, using the [mrsdeploy](../r-reference/mrsdeploy/mrsdeploy-package.md) functions and an R console application such as Rgui.exe , to a Machine Learning Server with [operationalization](../r-reference/mrsdeploy/configureServicePool.md) enabled is needed, to run the following commands:
```

Comment:
typo:  should be "established connection"

Actually, what do you think about the  following rephrasing?  It's a little shorter, which helps with readability:

"Given an existing connection to a Machine Learning Server with [operationalization](../r-reference/mrsdeploy/configureServicePool.md) enabled, you can use [mrsdeploy](../r-reference/mrsdeploy/mrsdeploy-package.md) functions and an R console application such as Rgui.exe  to  run the following commands:"

Reply:
your option looks great Heidi! You probably have a lot more experience than me creating documentation (and speaking English as well) ;)

So, how do you want to proceed? are you going to modify the docs or do you prefer from me to create a new pull request or modify the one already created? (I don't know if it's possible ...)

Reply:
I think you just copy-paste my changes over yours, and then do another commit.  Your latest changes will show up in this PR.